cl/corpus_importer/management/commands/make_aws_manifest_files.py

```diff
@@ -389,14 +401,7 @@ def export_records_in_batches(
             },
         )
         logger.info(
-            "\rRetrieved {}/{}, ({:.0%}), last PK processed: {},".format(
-                record_count + records_processed,
-                total_number_of_records,
-                (record_count + records_processed)
-                * 1.0
-                / total_number_of_records,
-                last_pk,
-            )
+            f"\rRetrieved {record_count + records_processed}/{total_number_of_records}, ({(record_count + records_processed) * 1.0 / total_number_of_records:.0%}), last PK processed: {last_pk},"
```

Comment:
In general, according to the new lint rules, we want to use string interpolation with loggers instead of using `f-strings` or `.format()`.

So we should do something like this:

```
logger.info(
    "\rRetrieved %d/%d (%.0f%%), last PK processed: %s",
    record_count + records_processed,
    total_number_of_records,
    (record_count + records_processed) * 100.0 / total_number_of_records,
    last_pk,
)
```

Unfortunately, this is not caught by the linter yet.
https://github.com/freelawproject/courtlistener/pull/5543#discussion_r2109793768

Reply:
Thanks for the suggestion! I'll update the logger code.